content/en/docs/concepts/configuration/assign-pod-node.md

```diff
@@ -111,7 +111,8 @@ For example, `example.com.node-restriction.kubernetes.io/fips=true` or `example.
 `nodeSelector` provides a very simple way to constrain pods to nodes with particular labels. The affinity/anti-affinity
 feature, greatly expands the types of constraints you can express. The key enhancements are
 
-1. the language is more expressive (not just "AND or exact match")
+1. `nodeSelector` provides only exact matches that you can with a logical AND operation, whereas affinity /
+   anti-affinity is significantly more expressive
```

Comment:
@sftim, line 114, I find this sentence difficult to understand/parse.
Are these edits helpful:
The affinity/anti-affinity language is more expressive. The language offers more matching rules besides exact matches created with a logical AND operation.